src/plugins/discover/public/application/main/components/layout/discover_layout.test.tsx

```diff
@@ -174,6 +179,17 @@ describe('Discover component', () => {
     expect(component.find('[data-test-subj="discoverChartOptionsToggle"]').exists()).toBeTruthy();
   });
 
+  test('the saved search title h1 gains focus on navigate', () => {
```

Comment:
@kertal I added some functional tests for accessibility which make this test sort of redundant. Do you think I should rollback changes to this test file or keep it?

Reply:
I think it's fine to keep it as a unit test too